data/json/items/tools.json

```diff
@@ -1,4 +1,4 @@
-[
+﻿[
```

Comment:
Your editor added a BOM (http://en.wikipedia.org/wiki/Byte_Order_Mark), the json parser doesn't like them. Most editors can be instructed in the the options to not do that.